pkg/controller/quayecosystem/resources/deployments.go

```diff
@@ -93,6 +93,10 @@ func GetQuayConfigDeploymentDefinition(meta metav1.ObjectMeta, quayConfiguration
 	BuildQuayConfigResourceLabels(meta.Labels)
 
 	envVars := []corev1.EnvVar{
```

Comment:
This is required in the latest version of Quay, 3.2.2

Reply:
Is the value always `new-installation`?

Reply:
For any new instance of Quay, that is correct, the other possible options are, 

add-new-fields
backfill-then-read-only-new
remove-old-field

Considering what we're doing with the operator the new-installation makes the most sense since we will be deploying  a new instance of Quay, not modifying it.

Reply:
A new field on the `quay` resource (enum type) should be created with the possible options (default to `new-installation`)

Reply:
done